batches/3.cohort-batch-3.md

```diff
@@ -5,3 +5,13 @@ This file contains the basic student information about the Cohort Batch 3.
 Add your introduction after the line below.
 
 <hr />
+
+
+Name: Rishiwardhan<br>
+Email: leerishiwardhan@gmail.com<br>
+Short Bio: I am a school student and loves to learn more things.<br>
+Educational Qualification: n/a<br>
```

Comment:
Here in Educational Qualification either you can mention N/A (capital letters) or High School.

Reply:
ok mam